.ado/pipelines/templates/stages-loadtest-azure.yaml

```diff
@@ -162,8 +164,10 @@ stages:
           arguments:
             -loadTestId "$(loadTestId)" `
             -apiEndpoint "$(azureLoadTestDataPlaneURI)" `
+            -apiVersion "$(azureLoadTestApiVersion)" `
             -testFileName "$(System.DefaultWorkingDirectory)/src/testing/loadtest-azure/scripts/catalog-test.jmx" `
-            -verbose:$true
+            -verbose:$true `
```

Comment:
Is there a reason for these two arguments to use the `-arg:value` syntax, instead of `-arg value` like the others? Is it because they're boolean?

Reply:
I'd issues with bool parameters in the past. That's why I'm using `-arg:value` instead of `-arg value` for them.